Firebase/Core/FIROptions.m

```diff
@@ -125,10 +126,10 @@ + (void)initialize {
   NSString *xcodeVersion = info[@"DTXcodeBuild"];
   NSString *sdkVersion = info[@"DTSDKBuild"];
   if (xcodeVersion) {
-    [FIRApp registerLibrary:@"xcode" withVersion:xcodeVersion];
+    [FIRApp registerLibrary:nil withName:@"xcode" withVersion:xcodeVersion];
```

Comment:
After seeing a few more of these `registerLibrary:nil`, I'm wondering if it's worth having a separate call just to add to the `userAgent`? What do you think?